File: A16AdvancePatterns/InvertedRotatedHalfPiramid.java
package A16AdvancePatterns;

public class InvertedRotatedHalfPiramid {
    public static void HalfPiramid(int row, int col) {
        for (int i = 1; i <= row; i++) {
            for (int j = 1; j <= col - i; j++) {
                System.out.print(" ");
            }
            for (int k = 1; k <= i; k++) {
                System.out.print("*");
            }
            System.out.println(" ");
        }
    }

    public static void main(String args[]) {
        HalfPiramid(4, 4);
    }
}

File: A16AdvancePatterns/InvertedRotatedHalfPiramid.js
function InvertedRotatedHalfPiramid(row, col) {
  for (let i = 1; i <= row; i++) {
    let line = "";
    for (let j = 1; j <= col - i; j++) {
      line += " ";
    }
    for (let j = 1; j <= i; j++) {
      line += "*";
    }
    console.log(line);
  }
}

InvertedRotatedHalfPiramid(4, 4);
